pkg/webhooks/machine_webhook_test.go

```diff
@@ -1146,6 +1146,79 @@ func TestMachineCreation(t *testing.T) {
 			},
 			expectedError: "[providerSpec.cluster.type: Invalid value: \"\": cluster type must be one of name or uuid, providerSpec.image.type: Invalid value: \"\": image type must be one of name or uuid, providerSpec.subnets: Invalid value: \"null\": missing subnets: nodes may fail to start if no subnets are configured]",
 		},
+		{
```

Comment:
What about a case with multiple subnets, but that is accepted and doesn't produce an error?

Reply:
Added another subnet to the no error case "with nutanix and the required fields set".